### DISCUSSION TITLE
H5P in PubPub

### DISCUSSION THREAD

**Discussion by tosteiner** [knowledgefutures/pubpub#1636] [Show and tell]
Just thought I'd share our embed of [H5P elements](https://h5p.org/) in PubPub, which - similarly to [TimelineJS](https://github.com/pubpub/pubpub/discussions/1626) - works like a charm ❤

Example: https://copim.pubpub.org/pub/interactions-with-open-access-books-part-2-tools-and-technologies/#building-blocks-h5p

Might be interesting for OER and open access publishing initiatives alike ...

**Comment by jeffpooley** [comment]
This is so great, @tosteiner! Thanks for sharing.

**Reply by gabestein** [reply]
Agreed, this is terrific! We've been talking for years about trying to build a sort of gallery of common interactives we could easily support...and it looks like someone's gone ahead and done it for us. We'll look into integrating more closely so you don't need to find a place to host the iFrame.

**Reply by tosteiner** [reply]
oh, yes - H5P is doing really amazing stuff, and all open and with proper licensing - all to facilitate open reuse and remix. 😄 

And it'd be awesome to have H5P's editing capabilities & hosting of packages integrated with PubPub! H5Ps dev team at Joubel ( https://joubel.com/index.html#about-h5p-01 ), are wonderful folks to work with, and they're always open to new adventures!

**Reply by writingpoli** [reply]
Awesome! Thank you for sharing!

**Comment by writingpoli** [comment]
[Clickable covers.](https://www.sareview.org/pub/sar6videos/)

**Reply by tosteiner** [reply]
oh, brilliant - well done! ♥️ H5P in the wild 😁

**Reply by jeffpooley** [reply]
Agreed—so impressive! Can’t wait to dive into this stuff.
